Configure

```diff
@@ -1193,7 +1193,9 @@ else               { $no_user_defines=1;    }
 
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
 
-unless ($disabled{engine}) {
+if ($disabled{engine}) {
+    $disabled{afalg} = "no-engine";
+} else {
```

Comment:
Ah yes, I was looking for something like this but saw it handled differently. I'll change that; but can you first explain the dependencies between dynamic-engine, static-engine and engine to me, so that I understand better what I'm actually doing?